src/validate.py

```diff
@@ -1,43 +1,26 @@
 import logging
 import re
 import inspect
+import json
+import os
 
 from flask import request, abort, redirect
 from functools import wraps
 from language import Language, DEFAULT_LANGUAGE
+from datetime import date
 
 import config as config_util
 
-DEFAULT_YEAR = '2019'
+CONFIG_DIR = './config'
+
+DEFAULT_YEAR = str(date.today().year)
```

Comment:
Bit concerned about this. If next Almanac doesn't come out until Nov next year, will this stop working for first 11 months of 2020?

Also DEFAULT_YEAR is also used in main.py to handle redirect from / to /en/2019/.

Reply:
thanks for pointing to the edge case here, will cover that 

so what should be DEFAULT_YEAR, current year ?

Reply:
I think it should probably stay hardcoded. Even when 2020 is started to be released we probably want to continue defaulting to 2019 until launch day.

Reply:
It will default to the latest year among the list of json files.

Reply:
I'm not sure, I share @bazzadp's concerns with this implementation.